### PR TITLE
Fully Working on Django 4.0

### PR DESCRIPTION
**Pull request by mkalioby**
Fixed the Signals issue so the now working with Django 4.0+, please try to release on pip

### CODE REVIEW

File: oidc_provider/lib/endpoints/authorize.py [hunk 1]
```diff
@@ -112,25 +140,32 @@ def validate_params(self):
             raise AuthorizeError(self.params['redirect_uri'], 'invalid_scope', self.grant_type)
 
         # Nonce parameter validation.
-        if self.is_authentication and self.grant_type == 'implicit' and not self.params['nonce']:
-            raise AuthorizeError(self.params['redirect_uri'], 'invalid_request', self.grant_type)
+        #if self.is_authentication and self.grant_type == 'implicit' and not self.params['nonce']:
```
Comment (juanifioren) [inline oidc_provider/lib/endpoints/authorize.py:143]: Why this is commented out? 🤔

File: oidc_provider/lib/endpoints/authorize.py [hunk 2]
```diff
@@ -112,25 +140,32 @@ def validate_params(self):
...
+        # check if user is no user and prompt = none
+
+        if self.is_authentication and self.params['prompt'] == 'none' and not self.request.is_authenticated:
+            raise AuthorizeError(self.params["redirect_uri"],'login_required',self.grant_type)
 
         # Response type parameter validation.
         if self.is_authentication \
                 and self.params['response_type'] not in self.client.response_type_values():
-            raise AuthorizeError(self.params['redirect_uri'], 'invalid_request', self.grant_type)
+            raise AuthorizeError(self.params['redirect_uri'], 'in	login_requiredvalid_request', self.grant_type)
```
Comment (juanifioren) [inline oidc_provider/lib/endpoints/authorize.py:156]: Should be `invalid_request`

**Review by juanifioren — CHANGES_REQUESTED** [review]

File: oidc_provider/lib/endpoints/authorize.py (hunk 2 above)
Comment (mkalioby) [reply oidc_provider/lib/endpoints/authorize.py]: I moved the PR to an older commit with had support for form_post (PASSED Implicit Form Post Test Suite), acr_values and, Dsicovery passed CONFIG OP

**Comment by samtin0x** [comment]
Hey, can we merge this?

**Comment by juanifioren** [comment]
The PR has no description on what is doing or changing + code is missing quality (fix all flake8 hints first).

**Comment by juanifioren** [comment]
Release `v0.8.0` is supporting Django 4. Closing.

**Pull request closed without merging**